osmoutils/accum/accum_test.go

```diff
@@ -318,3 +321,439 @@ func (suite *AccumTestSuite) TestClaimRewards() {
 		})
 	}
 }
+
+func (suite *AccumTestSuite) TestAddToPosition() {
+	type testcase struct {
+		startingNumShares sdk.Dec
+		startingUnclaimedRewards sdk.DecCoins
+		newShares   sdk.Dec
+
+		// accumInit and accumDelta specify the initial accum value 
+		// and how much it has changed since the position being added 
+		// to was created
+		accumInit	sdk.DecCoins
+		accumDelta  sdk.DecCoins
```

Comment:
nit: this helps to distinguish between initialization test case values and values we use to validate the result of executing system under test

Note, if you decide to apply it, it's better to do so locally because more changes would be necessary
```suggestion
		expectedAccumDelta  sdk.DecCoins
```